OpenSim/Simulation/SimbodyEngine/PinJoint.h

```diff
@@ -42,35 +42,16 @@ are in the desired direction.
 \image html pinJoint.gif
 
  @author Ajay Seth
- @version 1.0
  */
 class OSIMSIMULATION_API PinJoint : public Joint {
 OpenSim_DECLARE_CONCRETE_OBJECT(PinJoint, Joint);
 
-private:
-    static const int _numMobilities = 1;
-//=============================================================================
-// DATA
-//=============================================================================
-protected:
-
-    /** Pin has no additional properties*/
+    /** Specify the Coordinate of the PinJoint */
+    CoordinateP rz{ constructCoordinate(Coordinate::MotionType::Rotational) };
```

Comment:
It's not clear to me whether these are supposed to be in public, private, or protected.